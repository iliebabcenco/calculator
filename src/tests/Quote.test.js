import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../components/Quote';

it('renders correctly to Quote page', () => {
  const tree = renderer
    .create(<Link to="/quote">Quote </Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
