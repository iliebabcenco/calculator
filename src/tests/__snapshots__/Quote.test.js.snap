// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly to Quote page 1`] = `
<div
  className="quote-div"
>
  <p
    className="quote-p"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
